src/AssetManager.php

```diff
@@ -244,6 +244,64 @@ public function registerJsFile(string $url, array $options = [], string $key = n
         $this->jsFiles[$key]['attributes'] = $options;
     }
 
+    /**
+     * Registers a JS string.
+     *
+     * This method should be used for simple registration of JS files. If you want to use features of
+     * {@see AssetManager} like appending timestamps to the URL and file publishing options, use {@see AssetBundle}
+     * and {@see registerAssetBundle()} instead.
+     *
+     * @param string $url the JS string to be registered.
+     * @param array $options the HTML attributes for the script tag. The following options are specially handled and
+     * are not treated as HTML attributes:
+     *
+     * - `position`: specifies where the JS script tag should be inserted in a page. The possible values are:
+     *     * {@see \Yiisoft\View\WebView::POSITION_HEAD} in the head section
+     *     * {@see \Yiisoft\View\WebView::POSITION_BEGIN} at the beginning of the body section
+     *     * {@see \Yiisoft\View\WebView::POSITION_END} at the end of the body section. This is the default value.
+     *
+     * @return void
```

Comment:
```suggestion
     * @param string|null $key the key that identifies the JS code block. If null, it will use $js as the key. If two JS code
     * blocks are registered with the same key, the latter will overwrite the former.
     * @return void
```

Reply:
I moved this `@param` a bit up, to play with its other `@param` friends.